src/Components/Books.js

```diff
@@ -0,0 +1,97 @@
+            <li>Remove</li>
+            <li>Edit</li>
+          </ul>
+        </div>
+        <div className={styles.status}>
+          <div className={styles.bookStatus}>
+            <div className={styles.OvalPercentage} />
+            <div className={styles.PercentageWraper}>
+              <p className={styles.Percentage}>64%</p>
+              <h3 className={styles.Completed}>Completed</h3>
+            </div>
+          </div>
+          <div className={styles.bookUpdate}>
+            <p className={styles.Current}>CURRENT CHAPTER</p>
+            <h3 className={styles.Chapter}>Chapter 17</h3>
+            <button className={styles.button} type="button">UPDATE PROGRESS</button>
+          </div>
+        </div>
+      </li>
+      <li className={styles.bookItem}>
+        <div className={styles.bookContent}>
+          <p className={styles.School}>Science Fiction</p>
+          <h2 className={styles.Title}>Dune</h2>
+          <p className={styles.SuzanneCollins}>Frank Herbert</p>
+          <ul className={styles.bookEditor}>
+            <li className={styles.Comments}>Comments</li>
+            <li>Remove</li>
+            <li>Edit</li>
+          </ul>
+        </div>
+        <div className={styles.status}>
+          <div className={styles.bookStatus}>
+            <div className={styles.OvalPercentage} />
+            <div className={styles.PercentageWraper}>
+              <p className={styles.Percentage}>8%</p>
+              <h3 className={styles.Completed}>Completed</h3>
+            </div>
+          </div>
+          <div className={styles.bookUpdate}>
+            <p className={styles.Current}>CURRENT CHAPTER</p>
+            <h3 className={styles.Chapter}>Chapter 3: &quot;A Lesson Learned&quot;</h3>
+            <button className={styles.button} type="button">UPDATE PROGRESS</button>
+          </div>
+        </div>
+      </li>
+      <li className={styles.bookItem}>
+        <div className={styles.bookContent}>
+          <p className={styles.School}>Economy</p>
+          <h2 className={styles.Title}>Capital in the Twenty-First Century</h2>
+          <p className={styles.SuzanneCollins}>Suzanne Collins</p>
+          <ul className={styles.bookEditor}>
+            <li className={styles.Comments}>Comments</li>
+            <li>Remove</li>
+            <li>Edit</li>
+          </ul>
+        </div>
+        <div className={styles.status}>
+          <div className={styles.bookStatus}>
+            <div className={styles.OvalPercentage} />
+            <div className={styles.PercentageWraper}>
+              <p className={styles.Percentage}>0%</p>
+              <h3 className={styles.Completed}>Completed</h3>
+            </div>
+          </div>
+          <div className={styles.bookUpdate}>
+            <p className={styles.Current}>CURRENT CHAPTER</p>
+            <h3 className={styles.Chapter}>Introduction</h3>
+            <button className={styles.button} type="button">UPDATE PROGRESS</button>
+          </div>
```

Comment:
- [ ] Based on the requirements, all these should be in a component called the `Book` component. The `Book` component should be a child component of the `Books` component and contain all the elements that represent a single book. So instead of hardcoding all the values, you need to specify the elements that should be in the book. 💻 

 In the Books component, you should create an array of these books, map through the array, render the Book component and pass props to the Book component. 👍 😎 You can read this [article](https://www.geeksforgeeks.org/how-to-map-data-into-components-using-reactjs/) to get a better understanding. 👍 😎

Reply:
Hi @Gambit142, Thank You! :hugs: